src/cmake/lib.cmake

```diff
@@ -0,0 +1,89 @@
+# vi:set sw=2 et:
```

Comment:
I think the plugin validation could move to here right?

Reply:
Yap, that's the plan. Just takes testing on all three platforms so I'll do this in another change.